### ISSUE TITLE
# MVP List

### ISSUE THREAD

**Issue by andy-h-2016** [annieyang1993/DoorDash#2]
- [x] Should have 7 MVPs.
    - [x] 3 of those are User Auth, Heroku, and Production README.
    - [x] The other 4 are from the [MVP List](https://github.com/appacademy/curriculum/blob/master/full-stack-project/proposal/mvp-list.md) or they have clarified them with you
- [x] Contains a description sentence of the app
- [x] Includes two to three detailed bullets on functionality and presentation of feature
- [x] At least one CRUD feature, which states what CRUD operations are planned (creation, reading, updating, deletion)
- [x] Estimates how long it will take the code each MVP
- [x] Correctly formatted
    - [x] MVPs are listed in an ordered list
    - [x] Each MVP is broken down into bullet points
## Comments

**Comment by andy-h-2016** [comment]
- Please refer to the [Bluebird MVP list](https://github.com/appacademy/bluebird/wiki/mvp-list) for an example of what details to include on the MVP List and how to format it.
- Also refer to the [MVP checklist for pre-approved projects](https://open.appacademy.io/learn/swe-campus-hybrid/full-stack-project/mvp-checklist); look for websites with similar functionality to yours and see what MVPs they have.
- Order your MVPs in the order you are going to create them. For example, can't have a map showing the restaurants until you have your restaurants built out.

Direct feedback on your MVPs
- MVP 1: Map --> Users able to search restaurants near certain location
  - Good. would save this as last because it depends on restaurants.
- MVP 2: Location tracker of customer based off browser IP
  - Would be interesting. Would save this as a bonus item.
- MVP 3: Users can order food from the menu of a restaurant and checkout
  - Good.
- MVP 4: rewards based off of order history
  - Reject. Not a current feature of the DoorDash app. Don't want you designing new features, not the point of this project. Focus should be making the website look and function as much to the original as possible. You will have your hands full with the other features without having to make decisions on what a new feature will be like.
- MVP 5: Restaurants tracking orders; completed vs new
  - Reject. Very interesting, however, this is not a feature that is visible to the DoorDash public. So you would have to figure out a separate user_type for restaurant owners as well as design how this feature would look like. Again, goal is to mimic a website that a recruiter has visibility on. Recommend this as a bonus feature after you finish your project. Poking around, it seems that DoorDash has an entirely different login just for restaurants. Have some ideas for a watered-down version once you finish your project.

Bottom line - would recommend you organize your MVPs in the following categories:
- Business/Menu CRUD; display menu on business show page
- Home/splash page showing available restaurants to choose from
- Order and checkout
- Map of some sort
- Plus your bonus items

You can mess with the ordering of the MVPs as you see fit, but think of how each MVP may depend on the ones before it.

**Comment by andy-h-2016** [comment]
MVP List is looking pretty good now! Couple tweaks on ordering/labeling:
- I think items 8 and 9 should be labeled as bonus.
- Hosting on Heroku should be after you create the map
- Production README should be at the very end.

**Comment by andy-h-2016** [comment]
looks good
